### DISCUSSION TITLE
Messages icon breaks the rules?

### DISCUSSION THREAD

**Discussion by shutupayouface** [mirrorkeydev/Appstract#18] [Q&A] [answered]
per the previous discussion about design ethos, i noticed the `messages` icon has a line that terminates at the edge of the blue shape.  should this be updated or is it an exception because two other lines don't stop there?

thanks!

![Screenshot_20201210-124305](https://user-images.githubusercontent.com/75501809/101828079-f064ed00-3ae5-11eb-9e23-f4801b1074dd.png)  ![Screen Shot 2020-12-10 at 12 44 12 PM](https://user-images.githubusercontent.com/75501809/101827805-97955480-3ae5-11eb-986d-babb7c72fe8f.png)

**Comment by mirrorkeydev** [comment, marked as answer]
Yup, you're completely right in that it doesn't really match the design ethos. That icon was one of the very first 10 icons I made, and I hadn't 100% had my design ethos figured out back then 😅 I've been meaning to update it, but other stuff kept taking priority :) I'll open an issue to take care of that! https://github.com/mirrorkeydev/Appstract/issues/20

**Reply by shutupayouface** [reply]
i figured as much! but i'm still learning and wanted to make sure i fully understood the design ethos as best as possible.  thank you.

p.s. i think `netflix` might be in the same boat.
